daemon/system-info.sh

```diff
@@ -191,7 +191,10 @@ if [ -n "${lscpu}" ] && lscpu >/dev/null 2>&1 ; then
         LCPU_COUNT="$(echo "${lscpu_output}" | grep "^CPU(s):" | cut -f 2 -d ':' | sed -e 's/^[[:space:]]*//' -e 's/[[:space:]]*$//')"
         CPU_VENDOR="$(echo "${lscpu_output}" | grep "^Vendor ID:" | cut -f 2 -d ':' | sed -e 's/^[[:space:]]*//' -e 's/[[:space:]]*$//')"
         CPU_MODEL="$(echo "${lscpu_output}" | grep "^Model name:" | cut -f 2 -d ':' | sed -e 's/^[[:space:]]*//' -e 's/[[:space:]]*$//')"
-        possible_cpu_freq="$(echo "${lscpu_output}" | grep -F "CPU max MHz:" | cut -f 2 -d ':' | sed -e 's/^[[:space:]]*//' -e 's/[[:space:]]*$//' | grep -o '^[0-9]*') MHz"
+        possible_cpu_freq="$(echo "${lscpu_output}" | grep -F "^CPU max MHz:" | cut -f 2 -d ':' | sed -e 's/^[[:space:]]*//' -e 's/[[:space:]]*$//' | grep -o '^[0-9]*') MHz"
+        if [ "${possible_cpu_freq}" = " MHz" ] ; then
+            possible_cpu_freq="$(echo "${lscpu_output}" | grep -F "^CPU MHz:" | cut -f 2 -d ':' | sed -e 's/^[[:space:]]*//' -e 's/[[:space:]]*$//' | grep -o '^[0-9]*') MHz"
+        fi
```

Comment:
Using `grep -F` here is an error. `-F` means a basic. Both of these will result in a non-match. What you want is either `-G` (_basic regex_) or `-E` (_extended regex_). You've used a `^...` in your pattern indicating a regex of some kind :)